.github/workflows/pr.yaml

```diff
@@ -44,6 +44,22 @@ jobs:
           do
             sleep 1
           done
+      - name: Deploy konk-service
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service
+      - name: Deploy konk-service-operator
+        timeout-minutes: 5
+        run: |
+          make deploy-konk-service-operator
+          make deploy-example-konk-service
+          until kubectl wait --timeout=3m --for=condition=ready pod -l app.kubernetes.io/component=apiservice,app.kubernetes.io/instance=example-konkservice
+          do
+            sleep 1
+          done
+          POD_NAME=$(kubectl get pod -o name | grep konkservice)
+          kubectl exec -it $POD_NAME -- kubectl get apiservice v1alpha1.test.infoblox.com
+          kubectl exec -it $POD_NAME -- kubectl get crd appconfigs.stable.example.com websites.extensions.example.com
```

Comment:
The order of operations here isn't being enforced unless the readiness probe of this pod is contingent on the kubectl apply operations being completed. What we have here is

1. pod starts up
In parallel, these are happening
2. kubectl apply/default readiness probe (10sec) marks pod ready/kubectl exec -- kubectl get crd ...


What we need is to identify when the kubectl apply has completed here.
1. pod starts up
2. kubectl apply
3. readiness probe based on result of kubectl apply being successful
4. kubectl exec -- kubectl get crd ...


An easy way to accomplish this is to write out a file after the script completes successfully. A readiness probe can cat this file to know the pod is ready. K8s docs has a useful example

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes
```
readinessProbe:
  exec:
    command:
    - cat
    - /tmp/healthy
  initialDelaySeconds: 5
  periodSeconds: 5
```

Reply:
Added Readiness Probe.